### ISSUE TITLE
Disappear icons on tap

### ISSUE THREAD

**Issue by dmitrygribenchuk** [organicmaps/organicmaps#5285]
**Describe the issue**
Disappear nearby icons on tap. To appear to need to zoom.

**Steps to reproduce**
1. Tap on the icon
2. Tap on a nearby icon

**Expected behavior**

**Screenshots**
https://github.com/organicmaps/organicmaps/assets/15670247/a20a965f-5671-490b-a88e-e18369f209a6

**System information:**

 - Operating system and its version: Android 12
 - Organic Maps version: 2023.06.04-13-Google-beta
 - Device Model: Xiaomi Redmi Note 8 Pro

**Comment by hoizan** [comment]
I guess similar to #4685, and collectively in #1109

**Comment by Jean-BaptisteC** [comment]
Can you retry on latest version, we have added a new level of zoom to increase visibility of POI

**Comment by hoizan** [comment]
Yes, the icons that disappeared when a neighbouring icon is selected will reappear when another icon or place, which is slightly away, is clicked  on. I tested a few clusters of icons which are fairly closed to each other.
